core/src/main/scala/kafka/admin/AdminClient.scala

```diff
@@ -114,6 +116,13 @@ class AdminClient(val time: Time,
     listAllGroupsFlattened.filter(_.protocolType == ConsumerProtocol.PROTOCOL_TYPE)
   }
 
+  def listGroupOffsets(groupId: String): Map[TopicPartition, PartitionData] = {
+    val coordinator = findCoordinator(groupId)
+    val responseBody = send(coordinator, ApiKeys.OFFSET_FETCH, OffsetFetchRequest.allPartitionsOffsetFetchRequest(groupId))
+    val response = responseBody.asInstanceOf[OffsetFetchResponse]
+    response.responseData().asScala.toMap
```

Comment:
Could you please clarify your first comment above? With this change, I can still see the `Error: Consumer group 'cgroup1' does not exist.` message if 1) the consumer group is never created. 2) the consumer group is created by its offsets are all expired.

Are you referring to double `findCoordinator(...)` calls in this use case (through [here](https://github.com/vahidhashemian/kafka/blob/498e8dff14fa1392dd70fcf0867496a2def993d2/core/src/main/scala/kafka/admin/ConsumerGroupCommand.scala#L399) and [here](https://github.com/vahidhashemian/kafka/blob/498e8dff14fa1392dd70fcf0867496a2def993d2/core/src/main/scala/kafka/admin/ConsumerGroupCommand.scala#L406))? If so, one improvement would be to somehow preserve the `coordinator` value found in `describeConsumerGroup(...)` for use in `listGroupOffsets(...)`. Please advise if I misunderstood the issue. Thanks.